_data/jobs.yml

```
@@ -1,29 +1,29 @@
-- title: Ruby Developer (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/202168?language=en
-  description: With your skills in Ruby, Rails, JavaScript, and PostgreSQL you are responsible for the development of our crucial in-house tools.
-  location: Bilbao
+- title: Frontend Developer (m/f/d)
+  link: https://epages-gmbh.jobs.personio.de/job/224109
+  description: With your sense for modern UI design and aesthetics you create the visual appearance of our products and drive the visual identity.
+  location: Hamburg
 
 - title: Senior Java Developer (m/f/d)
   link: https://epages-gmbh.jobs.personio.de/job/202115
   description: Bring your knowledge of Java and Kotlin to work on a microservice-based architecture for our new ecommerce platform.
   location: Hamburg
 
 - title: JavaScript Engineer (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/201909?language=en
+  link: https://epages-gmbh.jobs.personio.de/job/201909
   description: Build a responsive, modular, and highly user interactive web application with us that is easy to scale and to extend.
   location: Hamburg
 
-- title: Software Developer - 3rd Party Integration, REST API (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/202173?language=en
-  description: Develop connectors that integrate third-party systems, such as payment gateways or product portals, into our ecommerce platform.
-  location: Barcelona
+- title: Technical Writer - API Docs & Social Media (m/f/d)
+  link: https://epages-gmbh.jobs.personio.de/job/522686
+  description: Empower developers to create awesome apps by taking care of our API documentation and get creative by creating content for our social media channels.
```

Quite a lot of "create" / "creative" in here. Would slightly rephrase it. By using "docs" instead of "documentation" we would also end up with four lines for each job description :)

```suggestion
  description: Empower developers to create awesome apps by taking care of our API docs and fill our Social Media channels with creative content.
```

---

```suggestion
  description: Empower developers to create awesome apps by taking care of our API docs and get creative with our social media channels.
```

---

Fine for me :) But I'd go with "Social Media channels". Aldijana reviewed the English job description and as she is using it in there, it seems to be more correct?